tests/Driver/CoreDriverTest.php

```diff
@@ -65,7 +65,7 @@ public function testInterfaceMethods(\ReflectionMethod $method)
 
         $driver = $this->getMockForAbstractClass('Behat\Mink\Driver\CoreDriver');
 
-        $this->setExpectedException('Behat\Mink\Exception\UnsupportedDriverActionException');
+        $this->expectException('Behat\Mink\Exception\UnsupportedDriverActionException');
```

Comment:
1. Where `expectException` method is defined, PHPUnit or Mink?
2. If that is PHPUnit, then are you sure, that method is present in every PHPUnit version where test suite will be used?